tox.ini

```diff
@@ -4,7 +4,9 @@ envlist=py27,py35,py36,py37,py38
 [testenv:dev]
 commands =
 envdir = venv
-deps=pre-commit>=1.11.0
+deps=
+    pre-commit>=1.11.0
+    coverage
```

Comment:
I suggest we also run coverage on CI during `[testenv]`:

```
    coverage run -m pytest -v
    coverage report
```

(I can't place comments in that line because it is not part of the changed lines).

If we have a good coverage score, we might also add `--fail-under=X` to `coverage report` so it fails if the coverage falls below that threshold.

Reply:
Done. We currently have 94% total coverage so I set `--fail-under=94`